test/e2e/containers/prow-npd-e2e/Makefile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build the container which Prow uses to run NPD e2e tests.
+
+IMAGE=gcr.io/gke-prow/prow-npd-e2e:live
```

Comment:
@krzyzacy Hi Sen, I want this container to be used publicly in the public Prow job.
In that case, do you know where is the best GCR repo I should host this container image in? Thanks!